srtcore/core.h

```diff
@@ -741,6 +743,7 @@ class CUDT
     int m_iBandwidth;                            // Estimated bandwidth, number of packets per second
     int m_iRTT;                                  // RTT, in microseconds
     int m_iRTTVar;                               // RTT variance
+    bool m_bIsSmoothedRTTReset;                  // If smoothed RTT has been reset after initialization
```

Comment:
The name is a bit confusing. Had to read all places where it is used to understand its purpose.
If tracks if `m_iRTT` was set to some actual estimate (either upon ACK or ACKACK) or not yet.
So using the term "smoothed" is a bit misleading, as `m_iRTT` does not have the same prefix in its name.

Why not just use the existing variables?
```c++
const bool bIsInitialRTT = (rtt == INITIAL_RTT) && (rttvar == INITIAL_RTTVAR);
```

Or consider renaming to something like `m_bIsInitialRTTUsed = true`.